api/getPageContentByPathname/index.ts

```diff
@@ -1,9 +1,8 @@
-import { CONTENTFUL_API_URL, headers } from '../constants'
-import { GET_PAGE_CONTENT_BY_PATHNAME } from './constants'
+import { CONTENTFUL_API_URL, GET_PAGE_CONTENT_BY_PATHNAME, headers } from '../constants'
 import { HttpRequestMethod } from '../types'
 import type { PageContentProps } from '../../components/Main/PageContent'
 
-export default async function getPageContentByPathname(pathname: string): Promise<PageContentProps | undefined> {
```

Comment:
I deleted the union type here because it feels like I am using the type test database. If there is a record in the database that has pathname but doesn't contain content, then the record should be populated with content rather than solve it somewhere in the frontend.